### PR TITLE
Added Profile Links

### PR DESCRIPTION
**Pull request by arjunpandey2003**
Issue #138

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -441,7 +441,8 @@ <h1>Keynote Speakers</h1>
 											<div class="social-links vertical-align">
 												<a href="https://twitter.com/catallman" target="default"><i class="icon social_twitter"></i></a>
 												<a href="https://www.linkedin.com/in/catallman"><i class="icon social_linkedin"></i></a>
-                                                <a href="https://www.facebook.com/cat.allman"><i class="icon social_facebook"></i></a>
+                                                                                                <a href="https://www.facebook.com/cat.allman"><i class="icon social_facebook"></i></a>
+												<a href="geekfeminism.wikia.com/wiki/Cat_Allman"><i class="icon fa fa-external-link"></i></a>
```
Comment (akshatnitd) [inline index.html]: Add `http` before the link

File: index.html [hunk 2]
```diff
@@ -520,6 +522,7 @@ <h2>Mike McQuaid</h2>
 												<a href="https://twitter.com/bytebot" target="default"><i class="icon social_twitter"></i></a>
                         									<a href="https://www.instagram.com/bytebot/"><i class="icon social_instagram"></i></a>
 												<a href="https://my.linkedin.com/in/colincharles" target="default"><i class="icon social_linkedin"></i></a>
+												<a href="www.bytebot.net/"><i class="icon fa fa-external-link"></i></a>
```
Comment (akshatnitd) [inline index.html]: Add `http` before link

File: index.html [hunk 3]
```diff
@@ -816,9 +821,9 @@ <h1>Speakers</h1>
 											It's great to be at FOSSASIA in Singapore! Fantastic atmosphere with lots of friendly devs, designers and business folks.
 										</div>
 										<div class="social-links vertical-align">
-											<a href="#"><i class="icon fa fa-external-link"></i></a>
+											<a href="www.bernardleong.com/biography"><i class="icon fa fa-external-link"></i></a>
```
Comment (akshatnitd) [inline index.html]: Add `http` before link

**Review by akshatnitd — CHANGES_REQUESTED** [review]
Also add

File: index.html (hunk 3 above)
Comment (akshatnitd) [reply index.html]: @arjunpandey2003 You forgot to make change in this link. Also, please squash the commits

**Review by akshatnitd — CHANGES_REQUESTED** [review]

**Comment by arjunpandey2003** [comment]
@akshatnitd please review again

**Comment by arjunpandey2003** [comment]
@mariobehling please review

**Review by akshatnitd — CHANGES_REQUESTED** [review]
@arjunpandey2003 Please squash the commits. Refer this for help if needed, https://gist.github.com/patik/b8a9dc5cd356f9f6f980

**Comment by arjunpandey2003** [comment]
@akshatnitd i have made these changes on the online editor so how should i squash

**Review by isuruAb — COMMENTED** [review]
Please squash your commits

**Comment by arjunpandey2003** [comment]
@isuruAb  i have made these changes on the online editor so how should i squash

**Comment by arjunpandey2003** [comment]
@akshatnitd @isuruAb Squashed

**Comment by arjunpandey2003** [comment]
Please approve

**Review by akshatnitd — APPROVED** [review]
Great work! :smiley:
Merging :rocket:

**Comment by arjunpandey2003** [comment]
Thanks @akshatnitd !

**Pull request merged**